superset/views/core.py

```diff
@@ -2636,7 +2636,7 @@ def sqllab_viz(self):
         table.columns = cols
         table.metrics = [SqlMetric(metric_name="count", expression="count(*)")]
         db.session.commit()
-        return self.json_response(json.dumps({"table_id": table.id}))
+        return json_success(json.dumps({"table_id": table.id}))
```

Comment:
`BaseSupersetView.json_response` is used quite heavily in `views`. If the method produces erroneous responses I think this should rather be addressed in `views/base.py` where it is defined.
https://github.com/apache/incubator-superset/blob/4568b2a532069bdd5595c8f79a55d20e6326f9e4/superset/views/base.py#L158-L163

Reply:
This method should produce correct response if we pass pathon object.
In current implementation we are passing JSON.dump(obj) here.
Most of the method in core.py when returning JSON response is using json_sucess